kuflow-robotframework/KuFlow/keywords.py

```diff
@@ -84,6 +78,19 @@ def get_client(self) -> KuFlowRestClient:
     def get_instance(self) -> "Keywords":
         return self
 
+    @keyword(tags=("settings",))
+    def convert_to_recursive_dictionary(self, value: dict) -> dict:
```

Comment:
I would leave it 'convert_to_dictionary", but if you want to express that the transformation is recursively I think that is better name it "convert_to_dictionary_recursively"

Reply:
I prefer an explicit name